src/network/impl/dialer_impl.cpp

```diff
@@ -3,116 +3,142 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+#include <functional>
+#include <iostream>
+
 #include <libp2p/connection/stream.hpp>
 #include <libp2p/log/logger.hpp>
 #include <libp2p/network/impl/dialer_impl.hpp>
 
-#define TRACE_ENABLED 0
-#include <libp2p/common/trace.hpp>
-
 namespace libp2p::network {
 
   void DialerImpl::dial(const peer::PeerInfo &p, DialResultFunc cb,
                         std::chrono::milliseconds timeout) {
+    SL_TRACE(log_, "Dialing to {}", p.id.toBase58().substr(46));
     if (auto c = cmgr_->getBestConnectionForPeer(p.id); c != nullptr) {
       // we have connection to this peer
 
-      TRACE("reusing connection to peer {}", p.id.toBase58().substr(46));
-      scheduler_->schedule([cb{std::move(cb)}, c{std::move(c)}] () mutable {
-        cb(std::move(c));
-      });
+      SL_TRACE(log_, "Reusing connection to peer {}",
+               p.id.toBase58().substr(46));
+      scheduler_->schedule(
+          [cb{std::move(cb)}, c{std::move(c)}]() mutable { cb(std::move(c)); });
```

Comment:
Abreviations and unreasonale shor variables' name are bad practice... =(

Reply:
Agree. The code was just clang-formatted. I will leave this unchanged, due to c is a locally if-scoped variable defined at the line above and cb is our commonly used shortening for a callback.